styles/main.css

```diff
@@ -238,16 +238,17 @@ img {
 .table{
     border-collapse: collapse;
     background: rgba(0, 0, 0, 0.05);
+}

```

Comment:
The table only needs a border-collapse and background rule.  The rest of the declarations you had in here should apply to table CELLS